### PR TITLE
Minimal Webui for Raiden

### PR DESCRIPTION
**Pull request by agatsoh**

### CODE REVIEW

**Comment by Magn3s1um** [comment]
In my opinion, the WebServer should boot up with raiden.  Maybe add the hooks in the cli.py run method so that webserver starts with raiden, rather than have the user have to start the UI manually.

File: raiden/raidenwebui/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```
Comment (LefterisJP) [inline raiden/raidenwebui/.editorconfig]: @andrevmatos Remove this from here. We don't commit personal editor configurations in the repository.

**Comment by LefterisJP** [comment]
As soon as you can remove the editor config from the PR we can approve for merging as we discussed, and further work should be done in **smaller** and **self-contained** PRs.

File: raiden/raidenwebui/.editorconfig (hunk 1 above)
Comment (andrevmatos) [inline raiden/raidenwebui/.editorconfig]: Actually, it's not personal.. this config is default from all projects created with `@angular/cli ng` tool, but I will remove it.

Comment (LefterisJP) [inline raiden/raidenwebui/.editorconfig]: Is it needed? If it's needed for angular keep it. But it seems like something that does not need to be comitted.

Comment (andrevmatos) [inline raiden/raidenwebui/.editorconfig]: No, not required, just created automatically by the tool. Already removed and merged again current master for PR' merge.

**Comment by LefterisJP** [comment]
Approved, but you also have to make sure tests pass.

**Comment by andrevmatos** [comment]
@Magn3s1um regarding your comment, still not implemented yet, but as Angular4 compiles to just a bunch of static files, the idea is, in the future, not to start it as a standalone server, but to provide it's pre-compiled (in build/setup.py time) files through `index.html` directly on Raiden' server. 
@LefterisJP not sure the current status of test suite for Angular UI, but we'll need to add it as specific tasks for testing suite in the future. Currently, it shouldn't fail, as test coverage is very limited.

**Pull request merged**